ci-operator/step-registry/mtr/OWNERS

```diff
@@ -0,0 +1,17 @@
+approvers:
```

Comment:
The OWNERS here should be mainly product qe then probably 1 or 2 members from CSPI.

Reply:
We shoudl discuss this in our meeting today. How many CSPI members and who do we choose? The person who wrote the scenario?

Reply:
Sachin and Shveta have been included as owners